packages/nx/src/utils/package-manager.ts

```diff
@@ -314,12 +329,16 @@ export async function packageRegistryView(
   args: string
 ): Promise<string> {
   let pm = detectPackageManager();
-  if (pm === 'yarn') {
+  if (pm === 'yarn' || pm === 'bun') {
     /**
      * yarn has `yarn info` but it behaves differently than (p)npm,
      * which makes it's usage unreliable
      *
      * @see https://github.com/nrwl/nx/pull/9667#discussion_r842553994
+     *
+     * inregards to bun. Bun is highly modelled after yarn v1
+     * bun info is a "subcommand reserved for future use by Bun."
+     * currently nothing like npm view/list exists
```

Comment:
list exists

Reply:
I've changed the comment. The behaviour completely different

Reply:
Added a better comment here npm view and bun pm ls works completely differently. They are likley adding a bun pm view command :) - it was mentioned on discord by the team

Reply:
Is this new commend any better?

Reply:
Yup view is not supported. I was only talking about list. LGTM